distrho/src/DistrhoPluginAU.cpp

```diff
@@ -172,6 +164,12 @@ class PluginAU : public AUEffectBase
         return AUEffectBase::GetProperty (inID, inScope, inElement, outData);
     }
 
+    void SetParameter(AudioUnitParameterID paramID,
+                      AudioUnitParameterValue value) override
+    {
+        fPlugin.setParameterValue(paramID, (float)value);
```

Comment:
nice, it was this simple!

Reply:
NB: Parameter changes still dont work testing with 10.6.8